### ISSUE TITLE
Temporary extracted files not deleted after an emulator crash

### ISSUE THREAD

**Issue by lordtaki** [digitalLumberjack/recalbox-os#365]
When lauching a zipped rom fails /recalbox/share/extractions contains the extracted rom.
Not a big deal but can lead to a disk full on the long term.

Saw it after sega32x and lynx failures for example.

Maybe a cronjob at startup of the system to clean this folder?

**Comment by digitalLumberjack** [comment]
Yes we already have a cleanup script. You're right it could delete the files in this folder.
:+1:
